src/templates/landing-page.jsx

```diff
@@ -36,9 +44,9 @@ function Landing({ data, location }) {
       // only route given
       // not currently being used
       // navigate(`${i18n.language}/${route}`);
-    } else {
+    } else if (lang !== i18n.language) {
```

Comment:
Nice catch!

Reply:
I wish I caught it myself but it was airbnb eslint!